theme.json

```diff
@@ -140,7 +140,7 @@
 			"spacing": {
 				"small": "max(1.25rem, 5vw)",
 				"medium": "clamp(2rem, 8vw, calc(4 * var(--wp--style--block-gap)))",
-				"large": "8rem"
+				"large": "clamp(4rem, 10vw, 8rem)"
```

Comment:
I had to make this responsive in order to provide a better mobile appearance. All of our other values were responsive already, and it seems to work just fine in other contexts where this variable is used. 

Before|After
---|---
![Screen Shot 2021-11-16 at 15 37 07](https://user-images.githubusercontent.com/1202812/142061924-232deae6-c0bc-45e3-b08d-acc6ede6f7d8.png)|![Screen Shot 2021-11-16 at 15 36 47](https://user-images.githubusercontent.com/1202812/142061925-210c5d61-2091-4b70-9645-d5892c149883.png)